sky/shell/platform/mojo/platform_view_mojo.cc

```diff
@@ -147,5 +156,23 @@ void PlatformViewMojo::OnEvent(mojo::EventPtr event,
   callback.Run();
 }
 
+void PlatformViewMojo::Create(
+    mojo::ApplicationConnection* connection,
+    mojo::InterfaceRequest<::keyboard::KeyboardService> request) {
+
+  mojo::ServiceProviderPtr sp;
+  connector_->ConnectToApplication("mojo:keyboard", mojo::GetProxy(&sp), nullptr);
+
+#ifdef OS_LINUX
+  ::keyboard::KeyboardServiceFactoryPtr factory;
+  mojo::NativeViewportEventDispatcherPtr dispatcher;
+  sp->ConnectToService(::keyboard::KeyboardServiceFactory::Name_, mojo::GetProxy(&factory).PassMessagePipe());
+  factory->CreateKeyboardService(mojo::GetProxy(&dispatcher), request.Pass());
+  viewport_->SetKeyEventDispatcher(dispatcher.Pass());
+#else
+  sp->ConnectToService(::keyboard::KeyboardService::Name_, request.PassMessagePipe());
```

Comment:
We should add another helper to `connect.h` that does this work.  We should try to avoid having end-developer code need to reference `Name_` directly.